guides/source/action_view_overview.md

```diff
@@ -1230,7 +1230,7 @@ Creates a field set for grouping HTML form elements.
 Creates a file upload field.
 
 ```html+erb
-<%= form_tag({ action: "post" }, multipart: true) do %>
+<%= form_with url: { action: "post" }, multipart: true do %>
   <label for="file">File to Upload</label> <%= file_field_tag "file" %>
   <%= submit_tag %>
```

Comment:
Why not use `form.file_field :file` and `form.submit` here? Though I can't remember what `form.submit` would do without a model instance.

Reply:
This example is specifically for the `file_field_tag` helper.  I think it uses `submit_tag` just for consistency.